services/horizon/docker/Dockerfile.dev

```
@@ -1,4 +1,4 @@
-FROM golang:1.14-stretch AS builder
+FROM golang:1.16.2 AS builder
```

```suggestion
FROM golang:1.16.1 AS builder
```
SDF's build server currently uses 1.16.1. To allow reproducible builds let's stick to 1.16.1. And thanks for fixing this, we forgot to update this file.